repxe-host.sh

```diff
@@ -11,5 +11,5 @@ REPO_DIR="`dirname ${BASH_SOURCE[0]}`"
 cd $REPO_DIR
 bundle config --local PATH vendor/bundle
 bundle config --local DISABLE_SHARED_GEMS true
-bundle exec --keep-file-descriptors ./repxe_host.rb $*
+bundle exec --keep-file-descriptors $PWD/repxe_host.rb $*
```

Comment:
The bundler configuration file contains relative paths.  What happens when you run bundle exec with an absolute path after configuring relative paths everywhere else?